src/main/java/parking/service/TaskService.java

```diff
@@ -0,0 +1,56 @@
+package parking.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import parking.beans.document.Log;
+import parking.exceptions.ApplicationException;
+import parking.helper.ExceptionHandler;
+import parking.helper.ExceptionMessage;
+import parking.repositories.LogRepository;
+import parking.utils.ActionType;
+
+import javax.servlet.http.HttpServletRequest;
+import java.util.Date;
+import java.util.List;
+
+@Service
+public class TaskService {
+
+    @Autowired
+    private MailService mailService;
+    @Autowired
+    private ExceptionHandler exceptionHandler;
+    @Autowired
+    private LogRepository logRepository;
+
+    public void notifyAboutNewUsers(HttpServletRequest httpRequest) throws ApplicationException {
+
+        List<Log> logRecords = logRepository.findDailyConfirmations(ActionType.EMAIL_CONFIRMED, new Date());
+
+        int numberOfUsers = logRecords.size();
+        if (numberOfUsers > 0) {
```

Comment:
This part must be improved. It's hard to understand why you use switch inside if